api/src/db/migrations/v150_extract_business_data.test.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  generatev149Business,
+  generatev149LicenseDetails,
+  generatev149ProfileData,
+  generatev149UserData,
+  getRandomv149LicenseStatus,
+} from "@db/migrations/v149_add_environment_data";
+import { migrate_v149_to_v150 } from "@db/migrations/v150_extract_business_data";
+
+describe("v150 migration adds version field to businesses object", () => {
+  it("should upgrade v149 user by adding version field to businesses object", () => {
+    const id = "business-1";
+    const v149ProfileData = generatev149ProfileData({ industryId: "employment-agency" });
+    const v149Business = generatev149Business({
+      profileData: v149ProfileData,
+      licenseData: {
+        lastUpdatedISO: "some last iso",
+        licenses: {
+          ["Pharmacy-Pharmacy"]: generatev149LicenseDetails({
+            licenseStatus: getRandomv149LicenseStatus(),
+          }),
+        },
+      },
+      id,
+    });
+
+    const v149UserData = generatev149UserData({
```

Comment:
Updated it, but the current way we set up our migrations means we create these generators for every version. I think we should consider revisiting this approach and potentially focus on generating only the parts we need for each migration. This could help reduce unnecessary duplication and make the migration process more efficient. Thoughts? @jphechter. Point for us to discuss tomorrow.